service/healthcheck/healthcheck.go

```diff
@@ -0,0 +1,28 @@
+package healthcheck
+
+import (
+	"context"
+	"net/http"
+
+	health "github.com/ONSdigital/dp-healthcheck/healthcheck"
+
+	cognitoclient "github.com/ONSdigital/dp-identity-api/cognitoclient"
+	cognito "github.com/aws/aws-sdk-go/service/cognitoidentityprovider"
+)
+
+const CognitoHealthy  = "Cognito Healthy"
+
+func CognitoHealthCheck(cognitoClient cognitoclient.Client, userPoolID *string) health.Checker {
+	
+	return func(ctx context.Context, state *health.CheckState) error {
+		_, err := cognitoClient.DescribeUserPool(&cognito.DescribeUserPoolInput{UserPoolId: userPoolID})
+	
+		if err != nil {
```

Comment:
If its possible to get additional detail from the `err` it might be useful to log it. I'm thinking it might be useful if anyone has to investigate an unexpected issue - the more details available the better.